requirements.txt

```diff
@@ -6,9 +6,10 @@ cffi==1.9.1
 codecov==2.0.11
 coverage==4.0.3
 cryptography==2.5
+Cython==0.29.13
```

Comment:
Why is this needed?

Reply:
`pysam` doesn't pin its `Cython` dependency well so I figured it safest for us to pin ours (I did have some trouble with incorrect `Cython` versions coming in from `pip`)

Reply:
Pinning 3rd party dependencies is problematic because it can cause resolution conflicts with other packages and also, it is easy to forget to remove them if the parent package is removed.